python/paddle/fluid/tests/unittests/dygraph_to_static/test_origin_info.py

```diff
@@ -65,7 +65,7 @@ def set_test_func(self):
         self.func = simple_func
 
     def set_static_lineno(self):
-        self.static_abs_lineno_list = [2, 3, 4]
+        self.static_abs_lineno_list = [3, 4, 5]
```

Comment:
Because I added "import paddle" to transformed code (see my change in utils.py), reviewers please check whether my changes to origin info test is correct here.

Reply:
It's correct. LGTM.